src/utils/daos.c

```diff
@@ -786,6 +786,35 @@ obj_op_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+#define OCLASS_NAMES_LIST_SIZE 512
+
+static void
+print_oclass_names_list(FILE *stream)
+{
+	char *str;
+	size_t size = OCLASS_NAMES_LIST_SIZE, len;
+
+again:
+	str = malloc(size);
+	if (str == NULL) {
+		fprintf(stderr, "failed to malloc %zu bytes to gather oclass names list\n",
```

Comment:
(style)  line over 80 characters

Reply:
for easier browsing of source code, it is a good idea to keep full strings on sigle line instead to split.